crates/stackable-telemetry/src/instrumentation/axum/mod.rs

```diff
@@ -338,6 +404,7 @@ impl SpanExt for Span {
             otel.status_message = Empty,
             http.request.method = http_method,
             http.response.status_code = Empty,
+            http.route = Empty,
```

Comment:
**note (non-blocker)**: with the attribute keys, as soon as we have one with a Rust reserved word, we need to wrap them in quotes.

Eg: `network.type` (type being a reserved word).

With a bit of luck, something like https://github.com/tokio-rs/tracing/pull/2254 can be implemented soon so that we can consistently use the Semantic Convention consts in these span definitions as well as in the `record` functions.

Reply:
Yes, that feature would be really nice to have. I have planned to invest some time in the upstream project if needed.

Reply:
Added quotes in d41179b